doc/contrib/doc.rst

```diff
@@ -0,0 +1,38 @@
+.. _contribdoc:
+
+Writing documentation
+=====================
+
+The source for `the documentation <https://arbor.readthedocs.io>`__ is
+found in the ``/doc`` subdirectory, C++ examples in ``/example`` and
+Python examples in ``/python/example``. You can add yours in the same
+way you would contribute code, please see the “Github workflow” section.
+
+.. todo::
+    Very much in progress.
+
+.. _contribdoc-namingconventions:
+
+Labels and filename conventions
+-------------------------------
+
+Although it is not absolutely essential to do so, we try to keep to the following conventions
+for naming files and labels, with the goal of making it easy to construct one from the other
+such that you don't have to remember or look anything up. We try to cross-link where we can,
+which means we label where we can, which translates to a large number of labels.
+
+Wherever possible, observe:
+
+* file names: avoid underscores as much as possible. E.g. `cpp/cable_cell.rst` -> `cpp/cablecell.rst`.
+* page-level labels in each file: the path concatenated without spaces. E.g. `cpp/cablecell.rst` -> `cppcablecell`.
+* heading labels in a file: `pagelevel-sectionname`. Feel free to slugify long headings.
+E.g. the morphology section in the C++ cable cell docs could be `cppcablecell-morph`.
+
+.. _contribdoc-lang:
+
+Language
+--------
+
+Although the primary language of Arbor is C++, we use English for most of the non-code.
+If you need some help writing English, you can consult the European Unions
```

Comment:
I would change from `If you need help writing English...` to `If in doubt, we recommend following the European Union's English style guide...`.